### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by LowkeyCoding**
Added validation to reservation form and customer form

### CODE REVIEW

**Review by pimber — APPROVED** [review]
I have looked it through and i can't find any thing wrong

File: Flea/Migrations/BingoContextModelSnapshot.cs [hunk 1]
```diff
@@ -52,11 +52,18 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Name")
                         .IsRequired()
-                        .HasColumnType("text");
+                        .HasMaxLength(64)
+                        .HasColumnType("character varying(64)");
```
Comment (jonasKjellerup) [inline Flea/Migrations/BingoContextModelSnapshot.cs:56]: this should be `NVARCHAR`

**Review by jonasKjellerup — CHANGES_REQUESTED** [review]

File: Flea/Models/Customer.cs [hunk 2]
```diff
@@ -27,11 +33,23 @@ public Customer(int id) : this(string.Empty, string.Empty)
...
 		
 		DbSet<Customer> IModelEntity<Customer, BingoContext>.GetDbSet(BingoContext ctx) => ctx.Customers!;
 	}
+	
+	public class ValidCustomer : ValidationAttribute
+	{
+
+		protected override ValidationResult IsValid(object? value, ValidationContext validationContext)
+		{
+			return (value is Customer {Id: <= 0} ? new ValidationResult(ErrorMessage) : ValidationResult.Success)
```
[the remark this answers is not in the document]
Comment (LowkeyCoding) [reply Flea/Models/Customer.cs]: så får du en switch expression

**Review by jonasKjellerup — APPROVED** [review]
This be good

**Pull request merged**